starobject.cpp: In member function 'void StarObject::getIndexCoords(KSNumbers*, double*, double*)':
starobject.cpp:285:21: error: 'isnan' was not declared in this scope

--- origsrc/kstars-4.11.5/kstars/dms.h	2014-01-02 13:30:05.000000000 -0600
+++ src/kstars-4.11.5/kstars/dms.h	2014-02-27 01:30:55.680001000 -0600
@@ -22,7 +22,7 @@
 
 #include <QString>
 
-#include <cmath>
+#include <math.h>
 #include "nan.h"
 
 /**@class dms
--- origsrc/kstars-4.11.5/kstars/skyobjects/starobject.cpp	2014-01-02 13:30:05.000000000 -0600
+++ src/kstars-4.11.5/kstars/skyobjects/starobject.cpp	2014-02-27 01:31:19.764378500 -0600
@@ -42,7 +42,7 @@
 //QVector<SkyPoint *> StarObject::Trail;
 // END DEBUG
 
-#include <cmath>
+#include <math.h>
 
 //----- Static Methods -----
 //
